### ISSUE TITLE
FNX-3448 ⁃ Remove "Library" and move items

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#10343]
Fenix "Library" is currently only accessible through the browser toolbar menu (not from the homescreen) and it only contains 2 items: Bookmarks and History

We should revisit this and determine whether we want to keep "Library" and add more items to it, or get rid of it.

**Comment by yoasif** [comment]
I vote for getting rid of it -- it doesn't work very well in Firefox desktop either as a concept, and that feels to me one of the only reasons to maintain it in Fenix (consistency/familiarity).

**Comment by lime124** [comment]
@topotropic

**Comment by topotropic** [comment]
Since we only have two items in the library (bookmarks, history) atm and we link to both directly from the home menu anyway, I propose to remove the library screen and update the site menu to reflect this.

Todo's:
- remove "Library" header in home menu
- remove "Library" entry from site menu
- add "Bookmarks" and "History" to site menu 

For bottom app bar: https://share.goabstract.com/6a1bcc22-bd3a-497a-bf7b-39ec63dd2ea5?mode=design&sha=5003d5fe9c05859083b502887be98b883e93a3af
Top app bar: https://share.goabstract.com/21538152-fa26-4a7a-95d7-a75d220d1b25?mode=design&sha=5003d5fe9c05859083b502887be98b883e93a3af

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build 200512 from 5/12** using the following devices:
• Google Pixel 3a (Android 10)
• Huawei Mate 20 Lite (Android 9)
• OnePlus A3 (Android 6.0.1)

► Screenshots
![Screenshot_20200512-145010](https://user-images.githubusercontent.com/51314259/81687632-cec55980-9461-11ea-8352-9429eef74c31.png)

✔️ Performed smoke tests on both Bookmarks and History sections and **haven't found any regressions**
✔️ Checked the telemetry ( ⋮ Main menu event type pings )

► **Bookmarks**
```
"events": [
        {
          "category": "events",
          "extra": {
            "item": "bookmarks"
          },
          "name": "browser_menu_action"
```

► **History**
```
"events": [
        {
          "category": "events",
          "extra": {
            "item": "history"
          },
          "name": "browser_menu_action"
```

**Comment by softvision-miralobontiu** [comment]
Hi,
This is also verified as fixed on Beta 5.1.0-beta.1 with Samsung Galaxy Tab S6 (Android 9), and Sony Xperia Z5 Premium (Android 7.1.1).
Thanks!
